app/views/admin/shared/_user_search.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="small-12 medium-6">
   <%= form_for(User.new, url: url, as: :user, method: :get) do |f| %>
     <div class="input-group">
-        <%= text_field_tag :name_or_email, "", placeholder: t("admin.shared.user_search.placeholder") %>
+        <%= text_field_tag :name_or_email, params[:name_or_email], placeholder: t("admin.shared.user_search.placeholder") %>
```

Comment:
How about doing this change in a different commit and adding a test for it? Right now I think all tests pass without this change :thinking:.